tpp/src/http/AuthHttpClient.js

```diff
@@ -125,6 +125,26 @@ class AuthHttpClient extends CoreAuthHttpClient {
         return this._instance(request);
     }
 
+    /**
+     * Sets destination account for once if it has'nt been set.
+     *
+     * @param {string} tokenRequestId - tokenRequestId
+     * @param {Array} transferDestinations - transferDestinations
+     * @return {Promise} response to the API call
+     */
+    async setTokenRequestTransferDestinations(tokenRequestId, transferDestinations) {
+        const req = {
+            tokenRequestId,
+            transferDestinations,
+        };
+        const request = {
+            method: 'put',
+            url: `/token-requests/set/transfer-destinations/${tokenRequestId}`,
```

Comment:
Sorry, I modified the Url later and forgot to share the change. Its over here > https://github.com/tokenio/lib-proto/commit/d3f3c1fd316cc33c36f8dcba810ad13daa20afa7